### PR TITLE
Remove complete upload icon in Edge

### PR DESCRIPTION
**Pull request by dkonopka**
### Suggested merge commit message ([convention](https://github.com/ckeditor/ckeditor5-design/wiki/Git-commit-message-convention))

Fix: Complete upload icon should not be rendered in Edge. Closes ckeditor/ckeditor5/issues/1066.

### CODE REVIEW

**Review by Mgsy — APPROVED** [review]
Works fine 👌

File: tests/imageupload/imageuploadprogress.js [hunk 1]
```diff
@@ -263,4 +264,25 @@ describe( 'ImageUploadProgress', () => {
 			'</figure>]'
 		);
 	} );
+
+	it( 'should not create completeIcon element when browser is Microsoft Edge', () => {
+		env.isEdge = true;
```
Comment (oskarwrobel) [inline tests/imageupload/imageuploadprogress.js]: You should use `sino#stub`. And you should restore the stub after the test. You can use `testUtils` sandbox (it is automatically restored in `aftearEach`).

```js
testUtils.sinon.stub( env, 'isEdge' ).returns( true );
```

**Comment by oskarwrobel** [comment]
CC is decreased https://github.com/ckeditor/ckeditor5-image/pull/220#issuecomment-403388338
image

File: src/imageupload/imageuploadprogress.js [hunk 2]
```diff
@@ -209,11 +210,14 @@ function _hideProgressBar( viewFigure, writer ) {
 function _showCompleteIcon( viewFigure, writer, view ) {
 	const completeIcon = new UIElement( 'div', { class: 'ck-image-upload-complete-icon' } );
 
-	writer.insert( ViewPosition.createAt( viewFigure, 'end' ), completeIcon );
+	// Because in Edge there is no way to show fancy animation of completeIcon we need to skip it.
+	if ( !env.isEdge ) {
```
Comment (oskarwrobel) [inline src/imageupload/imageuploadprogress.js]: I would move this check here https://github.com/ckeditor/ckeditor5-image/pull/220/files#diff-3efbae15fe72a5fbcecff93633200c17R108

**Comment by oskarwrobel** [comment]
CI is failing because of https://github.com/ckeditor/ckeditor5-image/issues/221.

**Pull request merged**